pom.xml

```diff
@@ -1090,6 +1111,58 @@
                 <version>${spring.version}</version>
             </dependency>
 
+            <dependency>
+                <artifactId>spring-expression</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <artifactId>spring-test</artifactId>
+                <groupId>org.springframework</groupId>
+                <version>${spring.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-test</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.apache.solr</groupId>
+                <artifactId>solr-cell</artifactId>
+                <version>${solr.client.version}</version>
```

Comment:
Again, I think this should be test scope.  You've copied all these dependences from `dspace-server-webapp` but seem to have changed the scope.

Reply:
Adding test scope here the code, because dspace-api has solr-cell as a regular (scope: compile) dependency, but adding scope: optional in the dependency management forces the dependency scope to optional everywhere in the project, so also in dspace-api.

Reply:
Adding test scope here the code, because dspace-api has solr-cell as a regular (scope: compile) dependency, but adding scope: optional in the dependency management forces the dependency scope to optional everywhere in the project, so also in dspace-api.

Reply:
Thanks for that explanation. This is fine "as-is" then.